pkg/controller/controller.go

```diff
@@ -533,6 +554,36 @@ func (ctrl *resizeController) patchClaim(oldPVC, newPVC *v1.PersistentVolumeClai
 	return updatedClaim, nil
 }
 
+/*
+This portion can be removed once PR in kubernetes repo containing change in resize_util.go is merged
+*/
+func (ctrl *resizeController) addPreResizeCapAnnotation(pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) error {
+	// if the pv already has a resize annotation skip the entire process
+	if metav1.HasAnnotation(pv.ObjectMeta, util.AnnPreResizeCapacity) {
+		return nil
+	}
+	pvClone := pv.DeepCopy()
+	qty := pvc.Status.Capacity[v1.ResourceStorage]
+	pvClone.ObjectMeta.Annotations[util.AnnPreResizeCapacity] = (&qty).String()
+	_, err := ctrl.patchPersistentVolume(pv, pvClone)
+	return err
+}
+
+/*
+This portion can be removed once PR in kubernetes repo containing change in resize_util.go is merged
+*/
+func (ctrl *resizeController) deletePreResizeCapAnnotation(pv *v1.PersistentVolume) error {
+	// if the pv does not have a resize annotation skip the entire process
+	if !metav1.HasAnnotation(pv.ObjectMeta, util.AnnPreResizeCapacity) {
+		return nil
+	}
+	pvClone := pv.DeepCopy()
+	delete(pvClone.ObjectMeta.Annotations, util.AnnPreResizeCapacity)
+
+	_, err := ctrl.patchPersistentVolume(pv, pvClone)
+	return err
+}
+
```

Comment:
I do not understand above comments. How will https://github.com/sunpa93/kubernetes/pull/1/files change enable these functions to be removed?

Reply:
I thought I got rid of the remaining comments from the previous commits used for the previous designs. These comments are no longer relevant. Please disregard them. I will remove them.